### PR TITLE
Nerfs public autolathes

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

I was going to add a whole DESIGN FLAG (!!! WOW!!!) to flag things but I realized I can just

hackable = FALSE

easy

## Why It's Good For The Game

i'm stubborn about public lathes but i'm perfectly fine with most of the hacked designs not being available without asking. i'm kinda worried this will turn into "secure the good stuff behind hack barrier" but honestly why would i be worried that works for the most part

oh and you can print this special secure autolathe board too and stuff yeah.

## Changelog
:cl:
balance: Public autolathes can no longer be hacked.
/:cl:

### CODE REVIEW

**Comment by Tupinambis** [comment]
Honestly I think this is a really silly way to handle it.

I'd rather either:
A: Remove public autolathes
B: change what public autolathes can print to begin with (Only very basic stuff that other departments pretty much need to function)
C: Nerf all lathes in this fashion.

Making them unhackable I think will just confuse people and honestly feels like an icky hugbox approach.

**Comment by silicons** [comment]
b is exactly what i am doing only not that hugboxed aka not basic items only
c is absolutely not; autolathes are fine at the moment.

File: code/game/machinery/autolathe.dm [hunk 1]
```diff
@@ -371,18 +372,25 @@
...
 /obj/machinery/autolathe/proc/adjust_hacked(state)
 	hacked = state
-	for(var/id in SSresearch.techweb_designs)
-		var/datum/design/D = SSresearch.techweb_design_by_id(id)
-		if((D.build_type & AUTOLATHE) && ("hacked" in D.category))
-			if(hacked)
-				stored_research.add_design(D)
-			else
-				stored_research.remove_design(D)
+	if(hackable || !hacked)
```
Comment (Ghommie) [inline code/game/machinery/autolathe.dm]: Early return if not hackable.

File: code/modules/research/designs/machine_desings/machine_designs_all_misc.dm [hunk 2]
```diff
@@ -26,6 +26,14 @@
 	category = list ("Misc. Machinery")
 	departmental_flags = DEPARTMENTAL_FLAG_ALL
 
+/datum/design/board/autolathe_secure
```
Comment (Ghommie) [inline code/modules/research/designs/machine_desings/machine_designs_all_misc.dm:29]: Unused design, please remove.

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -190,7 +190,7 @@
 	description = "A refresher course on modern engineering technology."
 	prereq_ids = list("base")
 	design_ids = list("solarcontrol", "recharger", "powermonitor", "rped", "pacman", "adv_capacitor", "adv_scanning", "emitter", "high_cell", "adv_matter_bin",
-	"atmosalerts", "atmos_control", "recycler", "autolathe", "high_micro_laser", "nano_mani", "mesons", "thermomachine", "rad_collector", "tesla_coil", "grounding_rod",
+	"atmosalerts", "atmos_control", "recycler", "autolathe", "secure_autolathe", "high_micro_laser", "nano_mani", "mesons", "thermomachine", "rad_collector", "tesla_coil", "grounding_rod",
```
Comment (Ghommie) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	"atmosalerts", "atmos_control", "recycler", "autolathe", "autolathe_secure", "high_micro_laser", "nano_mani", "mesons", "thermomachine", "rad_collector", "tesla_coil", "grounding_rod",
```

**Review by Ghommie — APPROVED** [review]

**Pull request merged**